src/index.ts

```diff
@@ -247,10 +247,22 @@ export default class Collection<Resource extends { [key: string]: any }> {
     });
   }
 
+  public viewAsArray(ids?: string[]): Resource[] {
+    return this.view(ids).getOrElse([]);
+  }
+
+  public viewAtAsArray(at: string): Resource[] {
```

Comment:
Is this naming consistent with the naming in other places in this repo?

Reply:
Not sure what you mean? This is the first place we're doing this, so we have `asNullable` (which is a current naming scheme in fp-ts and RemoteData), and now `viewAsArray` and `viewAtAsArray`.

Reply:
I mean the `viewAt` name, I don't think it's super clear what that means, but if it's consistent within the library I don't see any reason to change it

Reply:
Oh, yeah, we added that back when we did the parent stuff together. See #7

Reply:
yeah, remind me again - what is `at`?

Reply:
not trying to blow out the scope of these small changes but i think some/most/all of these methods could do with a little documentation for the layfolks like me who don't know these terms

Reply:
> not trying to blow out the scope of these small changes but i think some/most/all of these methods could do with a little documentation for the layfolks like me who don't know these terms

Agreed. I'll add both JSDoc style comments and some documentation to the README. That's definitely in the scope of what I'm trying to do!

Reply:
@dylanpyle 
> yeah, remind me again - what is `at`?

`at` is the concept of "parenting" that we _were_ using `EntityStoreById` for in our code. We noticed that `EntityStoreById` doesn't actually normalize the data between the different parents, even if they are the same resource, so we added it as a first-class concept to the `RemoteCollection`.

Maybe instead of the `view(id[])` concept we should keep `view()` meaning 'show me everything' and change `view(parentId)` to be 'show me the list at some parent ID key'? We would need to add an optional `parentId` to the `with*`/`with*List` etc.

Reply:
To be clear, we will be swapping out `EntityStoreById` everywhere in our code to use `RemoteCollection` so it's valuable to get this correct and intuitive now.

Reply:
I think I like it explicit more than an implicit optional argument

Reply:
@skswanke 
There already is an implicit optional argument `ids?: string[]`, we just never use it. It allows us to store a list of IDs that is some subset of the collection and get the `RemoteData<string[], Resource[]>` with those resources that exist. Idea there being pagination or something. However, we could just use the parenting to store those arrays of IDs and pass in the key instead of the actual `string[]` of IDs.

I'm kind of divided here. I think _less_ flexibility is what we want, but I don't want to remove legitimate use-cases.